INSTALL.rst

```diff
@@ -1,6 +1,249 @@
+
+is ``/usr/bin/python3``. When running the test command above, you may print
+the executable used to run python with the following python code:
+
+.. code-block:: python
+
+  import sys
+  print(sys.executable)
+  import skimage
+  print("scikit-image installed with version " + skimage.__version__)
+
+
+2. User local packages: Packages installed by your user account (using pip) may
+   take precedence over packages installed by the operating system.
+   If you have trouble getting the correct version of ``scikit-image``
+   take a look at the contents of the directories (if they exist):
+     - ``~/.local/lib/python3.6``
+     - ``~/.local/lib/python3.7``
+     - ``~/.local/lib/python3.8``
+   If you are trying to use the operating system package manager, we recommend
+   you simply delete all directories listed above.
+
+
+Supported platforms
+-------------------
+
+The platforms officially supported by scikit-image are:
+
+1. Windows 64 bit
+2. Mac OSX
+3. Linux 64 bit
+
+Since Windows 32 bit remains a popular platform, we continue to support it for
+the time being.  However, we strongly recommend all users of Windows 32 bit
+begin to switch over to Windows 64 bit if they can.
+
+Help us learn what other platforms you would like to install scikit-image on.
+We are very insterested in learning how ``scikit-image`` is
+`used <https://github.com/scikit-image/scikit-image/issues/4375>`_.
+
+Unsupported platforms include:
+
+1. Linux 32 bit.
+2. Raspberry Pi running Rapsbian (32 bit arm): While we do not official support
+   this distribution, we point users to `piwheels`_ and scikit-image's specific
+   page:
+     * https://www.piwheels.org/project/scikit-image/
+     * You may need to install additional system dependencies listed in
+       https://www.piwheels.org/project/imagecodecs/ See
+       `issue 4721 <https://github.com/scikit-image/scikit-image/issues/4721>_`.
+3. NVidia Jetson (64 bit arm):
+     * Follow conversation on
+       `Issue 4705 <https://github.com/scikit-image/scikit-image/issues/4705>`_.
+
+While we do not directly support the platforms above, many of the core
+developers have experience using the platforms listed above. Do not hesitate to
+ask us questions pertaining to your specific use case.
+
+If you would like to help package scikit-image for the platforms above, reach
+out on github so that we may help you get started.
+
+
+End of instructions
+===================
+
+I propose ending the installation instructions here. This should get everybody
+a working installation of scikit-image without complicating things with:
+
+- compiler requirements (which users don't need)
+- Build and doc dependencies
+- Conda-forge instructions which we don't test with CIs and go stale quickly (e.g. channel priority)
+  - Almost every month, I get a user confused about how to install opencv on conda-forge
+    due to confusing instructions specifying the ``-c conda-forge``. I consider
+    conda-forge rather advanced usage.
```

Comment:
Shouldn't this rather go into the PR description? Almost missed this relevant info to a comment I made.  :sweat_smile:

Reply:
moving.